test/test_unary_ufuncs.py

```diff
@@ -489,7 +489,10 @@ def _test_out_arg(self, op, input, output, expected, **kwargs):
         else:
             res = op(input, out=output, **kwargs)
             self.assertTrue(res is output)
-            self.assertEqual(output, expected.to(output.dtype))
+            if op.name == "sinc":
+                self.assertEqual(output, expected.to(output.dtype), atol = 1e-6, rtol = 1e-6)
+            else:
+                self.assertEqual(output, expected.to(output.dtype))
```

Comment:
I'm concerned you are altering the precision without considering the dtype.

Default precisions are found here:

https://github.com/pytorch/pytorch/blob/a84ed8be6d2abd79d3867dfe02563a829919ea9e/torch/testing/_internal/common_utils.py#L1777-L1788

Reply:
I think what you're really looking for is the precision override where the 'sinc' unary op is defined.

https://github.com/pytorch/pytorch/blob/a84ed8be6d2abd79d3867dfe02563a829919ea9e/torch/testing/_internal/common_methods_invocations.py#L11627-L11642

Reply:
Thanks @jeffdaily for the notes. Yes, let me define the precision override for the complex128 data type and test it out.

Reply:
The precisionOverride defined in https://github.com/pytorch/pytorch/blob/a84ed8be6d2abd79d3867dfe02563a829919ea9e/torch/testing/_internal/common_methods_invocations.py#L11627-L11642 does not help in this case, Its used for dtype precision (and not for atol and rtol).
I tried to use self.precision (obtained from precisionOverride ) in atol and rtol (test/test_unary_ufuncs.py), however it fails for other non sinc test because atol and rtol values are not defined over there.